include/antidote.hrl

```diff
@@ -230,6 +233,6 @@
 %% clocksi_read_item_fsm as pointers to the materializer's ets caches
 -record(mat_state, {
 	  partition :: partition_id(),
-	  ops_cache :: cache_id(),
-	  snapshot_cache :: cache_id(),
-	  is_ready :: boolean()}).
+	  ops_cache :: atom() | cache_id(),
```

Comment:
Let's file an issue for that. Dynamic creation of atoms should be considered harmful and at least inspected with care.